package section3;


class MyException1 extends Exception{
	   String str1;
	   MyException1(String str2) {
		str1=str2;
	   }
	   public String toString(){ 
		return ("MyException Occurred: "+str1) ;
	   }
	}
	class implementtheexceptionhandle{
	   public static void main(String args[]){
		try{
			System.out.println("this is my first code");
			// I'm throwing the custom exception using throw
			throw new MyException1("This is first  Message");
		}
		catch(MyException1 exp){
			System.out.println("Catch Block") ;
			System.out.println(exp) ;
		}
	   }
	}